File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Precancel/Cancel_Refund.vb
Imports Newtonsoft.Json

Namespace ExpediaRapid.SerializableClasses.BookingItinerary
    Public Class CancelRefund

        <JsonProperty("amount")>
        Public Property Amount As String

        <JsonProperty("currency")>
        Public Property CurrencyCode As String

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Precancel/Cancel_Refund.cs
namespace iVectorOne.Suppliers.ExpediaRapid.SerializableClasses.BookingItinerary
{
    using Newtonsoft.Json;

    public class CancelRefund
    {

        [JsonProperty("amount")]
        public string Amount { get; set; }

        [JsonProperty("currency")]
        public string CurrencyCode { get; set; }

    }

}